src/main/java/persistence/model/EntityTable.java

```diff
@@ -36,7 +36,22 @@ public List<EntityColumn> getPrimaryColumns() {
         return tableColumns.getPrimaryColumns();
     }
 
+    public List<EntityColumn> getNonPrimaryColumns() {
+        return tableColumns.getNonPrimaryColumns();
+    }
+
     public void setColumns(List<EntityColumn> columns) {
         tableColumns.setColumns(columns);
     }
+
+    public List<EntityColumn> getInsertableColumns(EntityTable table) {
+        return table.isPrimaryColumnsValueSet()
+                ? table.getColumns()
+                : table.getNonPrimaryColumns();
+    }
+
+    public Boolean isPrimaryColumnsValueSet() {
+        return getPrimaryColumns().stream().allMatch(EntityColumn::isValueNotNull);
+    }
```

Comment:
객체 내부에서만 사용하는 메서드는 private 접근 제어자를 권장드립니다 :)

```suggestion
    private Boolean isPrimaryColumnsValueSet() {
        return getPrimaryColumns().stream().allMatch(EntityColumn::isValueNotNull);
    }
```